### PR TITLE
Filter dags by owner

### LINKED ISSUE #9102: Add ability to filter DAG list by clicking on dag owner
**Description**

Similar to https://github.com/apache/airflow/issues/8821

**Use case / motivation**

DAG list is organised by dag id so if someone want to see his dags he must search for it in the bar. it can good if by clicking on owner in owner column it will show only this person dags

@Sangarshanan what do you think?

### PR DESCRIPTION
**Pull request by Sangarshanan**
Filter DAGs by clicking on the dag owner

closes: #9102

---

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/www/templates/airflow/dags.html [hunk 1]
```diff
@@ -137,7 +137,11 @@ <h2>DAGs</h2>
 
             <!-- Column 5: Dag Owners -->
             <td>
-              {{ dag.owners }}
+              <a class="label label-default"
+                 href="?search={{ dag.owners }}"
+                 style="margin: 3px;">
+                 {{ dag.owners }}
```
Comment (mik-laj) [inline airflow/www/templates/airflow/dags.html]: I wonder if we should not display a separate link for each person.

Comment (Sangarshanan) [reply airflow/www/templates/airflow/dags.html]: How do you think we should do this ? Also why not links, I am asking because everything else is a link 😅

**Comment by kaxil** [comment]
Can you please rebase your PR on latest Master since we have applied [Black](https://github.com/apache/airflow/commit/4e8f9cc8d02b29c325b8a5a76b4837671bdf5f68) and [PyUpgrade](https://github.com/apache/airflow/commit/8c42cf1b00c90f0d7f11b8a3a455381de8e003c5) on Master.

It will help if your squash your commits into single commit first so that there are less conflicts.

File: airflow/www/templates/airflow/dags.html (hunk 1 above)
Comment (kaxil) [reply airflow/www/templates/airflow/dags.html]: ^^ @mik-laj

Comment (mik-laj) [reply airflow/www/templates/airflow/dags.html]:
````html
                  {% for owner in dag.owners.split(",") %}
                  <a class="label label-default"
                     href="?search={{ owner | trim }}"
                     style="margin: 3px;">
                     {{ owner | trim }}
                  </a>
                  {% endfor %}
````
Screenshot 2020-11-08 at 01 08 57

What do you think aboiut it?

Comment (mik-laj) [reply airflow/www/templates/airflow/dags.html]: Before:
Screenshot 2020-11-08 at 01 11 24

Comment (Sangarshanan) [reply airflow/www/templates/airflow/dags.html]: Ah yes I think I understood the question wrong, this does make more sense

**Review by mik-laj — APPROVED** [review]
LGTM, I would love to hear feedback from @ryanahamilton .

File: airflow/www/templates/airflow/dags.html [hunk 2]
```diff
@@ -125,7 +125,15 @@ <h2>DAGs</h2>
                   </div>
                 </td>
                 {# Column 3: Dag Owners #}
-                <td>{{ dag.owners }}</td>
+                <td>
+                  {% for owner in dag.owners.split(",") %}
+                  <a class="label label-default"
+                     href="?search={{ owner | trim }}"
+                     style="margin: 3px;">
```
Comment (ryanahamilton) [inline airflow/www/templates/airflow/dags.html]: The tiniest adjustment will keep the left side of this left-aligned column just right:
```suggestion
                     style="margin: 3px 6px 3px 0;">
```

**Review by ryanahamilton — APPROVED** [review]
One minuscule styling suggestion, but otherwise LGTM!

**Pull request merged**